lib/membrane/video_compositor/scene/object/input_video.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Membrane.VideoCompositor.Scene.Object.InputVideo do
+  @moduledoc """
+  Structure representing `InputVideo` wrapper for using input
+  Membrane Pad of an element as a object in a scene.
+  """
+  alias Membrane.Pad
+
+  @enforce_keys [:input_pad]
+  defstruct @enforce_keys
+
+  @typedoc """
+  Defines pad wrapper.
+  """
+  @type t :: %__MODULE__{
+          input_pad: Pad.ref_t()
+        }
```

Comment:
I don't know if we shouldn't hold the stream format here

Reply:
We shouldn't - input stream format can change with membrane callback, meaning we would need to enforce updating the scene with it, even when the VC output image doesn't change

Reply:
Because of that, binding the `InputVideo` output resolution to a certain value also doesn't seem like a good idea for me either.

Reply:
That's true, but this makes me a bit unsure about the way of handling stream format changes at the boundary between the queue and the compositor